Dockerfile

```diff
@@ -1,54 +1,87 @@
-FROM ubuntu:18.04
-
-RUN apt-get update && \
-    apt-get install -y curl \
-                       git \
-                       python \
-                       python-pip \
-                       python-dev \
-                       autoconf \
-                       libtool \
-                       gawk
-RUN curl -O https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz
-RUN tar xvf go1.10.3.linux-amd64.tar.gz
+# Build stage
+FROM golang:1-alpine as builder
 
 WORKDIR /app
 
-COPY ./bin /app/bin
-COPY ./lib /app/lib
-COPY ./share /app/share
-COPY ./requirements.txt /app
-COPY ./src/we-lang/we-lang.go /app
+COPY ./share/we-lang/we-lang.go /app
+
+RUN apk add --no-cache git
 
-# There are several files that must be fetched/created manually
-# before building the image
-COPY ./.wegorc /root
-COPY ./.ip2location.key /root
-COPY ./airports.dat /app
-COPY ./GeoLite2-City.mmdb /app
 
-RUN export PATH=$PATH:/go/bin && \ 
+RUN true && \ 
```

Comment:
The true is useless. It can be removed.

Reply:
I don't know why @elerch Emil added it; maybe just for the sake of some edit simplicity (similar to that as a comma added after the final element in a list)

Reply:
Yeah - that's exactly why I do that. ;-) Makes it easier to copy/paste between different Dockerfiles

Reply:
I knew that :) Is it an existing convention for Dockerfiles or is it your invention?

Reply:
It is nothing official and in my opinion just noise.